app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx

```diff
@@ -272,8 +272,10 @@ export const AppLayoutSidebarLaunched = ({
                       </LinkExternal>
                       .
                     </>
+                  ) : analysis?.metadata?.external_config?.skip ? (
```

Comment:
I added this [per this comment](https://github.com/mozilla/experimenter/issues/6200#issuecomment-901398847) in the issue. I believe it'll only affect [this experiment](https://experimenter.services.mozilla.com/nimbus/set-default-as-first-screen-100-roll-out) for now.